tests/unit/test_ringcells_reports.py

```diff
@@ -0,0 +1,196 @@
+from pathlib import Path
```

Comment:
I was thinking that maybe it would be useful to have a small intro that explains what these tests are about

Reply:
I've added the doctrings for the error handling tests.

Reply:
I meant a docstring at the beginning of the file just to give a rundown of what we want to achieve with this file

Reply:
so, no incipit to explain what we want to do with these tests?

Reply:
I thought the filename is self explanatory, plus each test function has its docstrings.